website/ember_osf_web/views.py

```diff
@@ -8,7 +8,7 @@
 
 from website.settings import EXTERNAL_EMBER_APPS, PROXY_EMBER_APPS, EXTERNAL_EMBER_SERVER_TIMEOUT
 
-ember_osf_web_dir = os.path.abspath(os.path.join(os.getcwd(), EXTERNAL_EMBER_APPS['ember_osf_web']['path']))
+ember_osf_web_dir = os.path.abspath(os.getcwd() + '/website' + EXTERNAL_EMBER_APPS['ember_osf_web']['path'])
```

Comment:
This gives the ember_osf_web_dir a different value - `'/code/website/ember_osf_web'` instead of `'/ember_osf_web'`?   is this going to be an issue?

Reply:
@mfraezz - this is the one piece I'm not sure about

Reply:
BTW this is for Travis because of OS changes for them the route produced by `getcwd` changed.

Reply:
Would this make sense to turn into a setting and override it for travis?

Reply:
This shouldn't affect deployed environments, which have `PROXY_EMBER_APPS` set to `True`. However, it will likely affect local development. Rather than changing it here, why not change `EXTERNAL_EMBER_APPS['ember_osf_web']['path']`'s value in `local-travis.py`?

Should be using `os.path.join` rather than additive concatenation, too.